core/src/main/java/io/cucumber/core/stepexpression/DocStringType.java

```diff
@@ -0,0 +1,44 @@
+package io.cucumber.core.stepexpression;
+
+
+import java.lang.reflect.Type;
+
+public class DocStringType {
```

Comment:
```suggestion
@API(status = API.Status.STABLE)
public final class DocStringType {
```

And you may have to fix the import manually.